packages/maplestory-open-api-js/src/types/ranking.ts

```diff
@@ -0,0 +1,94 @@
+export interface CommonRankingProps {
+  /** 랭킹 업데이트 일자 (KST, 일 단위 데이터로 시, 분은 일괄 0으로 표기)
+   * example: 2023-12-21T00:00+09:00
+   * */
+  date: string;
+  /** 종합 랭킹 순위 */
+  ranking: number;
+  /** 캐릭터 명 */
+  character_name: string;
+  /** 월드 명 */
+  world_name: string;
+  /** 직업 명 */
+  class_name: string;
+  /** 전직 직업 명 */
+  sub_class_name: string;
+}
+
+export interface OverallRanking {
+  ranking: CommonRankingProps &
+    {
+      /** 캐릭터 레벨 */
+      character_level: number;
+      /** 캐릭터 경험치 */
+      character_exp: number;
+      /** 캐릭터 인기도 */
+      character_popularity: number;
+      /** 길드 명 */
+      character_guildname: string;
+    }[];
```

Comment:
`&` 이 정상적으로 사용되지 않고있습니다!

- 괄호로 우선순위 이용

```suggestion
  ranking: (CommonRankingProps &
    {
      /** 캐릭터 레벨 */
      character_level: number;
      /** 캐릭터 경험치 */
      character_exp: number;
      /** 캐릭터 인기도 */
      character_popularity: number;
      /** 길드 명 */
      character_guildname: string;
    })[];
```

- 제네릭 이용

```suggestion
  ranking: Array<CommonRankingProps &
    {
      /** 캐릭터 레벨 */
      character_level: number;
      /** 캐릭터 경험치 */
      character_exp: number;
      /** 캐릭터 인기도 */
      character_popularity: number;
      /** 길드 명 */
      character_guildname: string;
    }>;
```

Reply:
반영했습니다!